emissionsapi/openapi.yml

```diff
@@ -56,6 +63,20 @@ components:
         type: string
       description: "`country` will use a preselected rectangle containing the specified country. Like 'DE'."
       example: DE
+    polygon:
+      in: query
+      name: polygon
+      schema:
+        type: array
+        items:
+          type: number
+        minItems: 3
+      description: | 
+          `polygon` will let you define your own polygon:<br>
+          <p><em>lon1,lat1,lon2,lat2, ...</em></p>
+          The parameter can be concated by ',' e.g. "?polygon=(30,10,40,40,20,40,10,20,30,10)".<br>
+          If the polygon is not closed (the last lon/lat pair is not equal to the first lon/lat pair), the first pair will be added at the end of the array.
```

Comment:
First of all I would remove the inline HTML. Also there are several possibilities to serialize an array in the openapi specification, see https://swagger.io/docs/specification/describing-parameters/#query-parameters. So I would not try to describe them here (I think the one you used is even wrong :smile:). Also I find the text not very precise. What about something like

> The parameter must be in the form lo1,la1,lo2,la2,... and every pair of values represent a vertex of longitude and latitude.
> The `polygon` is then defined as the line segments of the consecutive vertices.
> 
> If the first and last vertices does not match, the first vertex is automatically added as the last vertex to close the polygon.

Reply:
Looks good, I was not happy with the phrase anyways...